client/src/components/Tool/ToolFooter.vue

```diff
@@ -1,88 +1,71 @@
-                        prefix="-"
-                    />
-                </div>
-                <div v-if="requirements && requirements.length > 0" class="metadata-section">
-                    <span class="metadata-key"
-                        >Requirements:
-                        <a href="https://galaxyproject.org/tools/requirements/" target="_blank">
-                            <font-awesome-icon
-                                v-b-tooltip.hover
-                                title="Learn more about Galaxy Requirements"
-                                icon="question"
-                            />
-                        </a>
-                    </span>
-                    <div v-for="(requirement, index) in requirements" :key="index">
-                        - {{ requirement.name }}
-                        <span v-if="requirement.version"> (Version {{ requirement.version }}) </span>
-                    </div>
-                </div>
-                <div class="metadata-section" v-if="license">
-                    <span class="metadata-key">License:</span>
-                    <License :licenseId="license" />
-                </div>
-                <div v-if="xrefs && xrefs.length > 0" class="metadata-section">
-                    <span class="metadata-key">References:</span>
-                    <div v-for="(xref, index) in xrefs" :key="index">
-                        - {{ xref.reftype }}:
-                        <template v-if="xref.reftype == 'bio.tools'">
-                            {{ xref.value }}
-                            (<a :href="`https://bio.tools/${xref.value}`" target="_blank"
-                                >bio.tools
-                                <font-awesome-icon
-                                    v-b-tooltip.hover
-                                    title="Visit bio.tools reference"
-                                    icon="external-link-alt"
-                                /> </a
-                            >) (<a :href="`https://openebench.bsc.es/tool/${xref.value}`" target="_blank"
-                                >OpenEBench
-                                <font-awesome-icon
-                                    v-b-tooltip.hover
-                                    title="Visit OpenEBench reference"
-                                    icon="external-link-alt"
-                                /> </a
-                            >)
-                        </template>
-                        <template v-else>
-                            {{ xref.value }}
-                        </template>
-                    </div>
-                </div>
-                <div v-if="creators && creators.length > 0" class="metadata-section">
-                    <span class="metadata-key">Creators:</span>
-                    <Creators :creators="creators" />
-                </div>
-            </b-card>
-        </b-collapse>
-    </div>
+    <b-card class="tool-footer" v-if="hasContent">
```

Comment:
Can you adjust the selenium test as appropriate then ?
It's looking for `collapse-about`